string.go

```diff
@@ -1,13 +1,13 @@
 package lo
 
 import (
+	"math"
+	rand2 "math/rand/v2"
 	"regexp"
 	"strings"
 	"unicode"
 	"unicode/utf8"
 
-	"github.com/samber/lo/internal/rand"
```

Comment:
Please use github.com/samber/lo/internal/rand instead of math/rand/v2

Reply:
ok